From febed0bc7a91798bfd7638083e95685370bfc443 Mon Sep 17 00:00:00 2001
From: Kevin McCarthy <kevin@8t8.us>
Date: Fri, 27 Mar 2020 15:13:15 -0700
Subject: Fix up mailboxes documentation synopsis

Specify the flags as type "option", instead of as parameters.

Upstream-commit: https://gitlab.com/muttmua/mutt/commit/febed0bc7a91798bfd7638083e95685370bfc443
Co-authored-by: Richard Russon <rich@flatcap.org>
---
 doc/manual.xml.head | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/doc/manual.xml.head b/doc/manual.xml.head
index 90e2dd90..c73206bb 100644
--- a/doc/manual.xml.head
+++ b/doc/manual.xml.head
@@ -3689,19 +3689,19 @@ to save read mail in more than a single mailbox).
 <arg choice="opt">
   <group choice="opt">
     <arg choice="plain">
-      <replaceable class="parameter">-poll</replaceable>
+      <option>-poll</option>
     </arg>
     <arg choice="plain">
-      <replaceable class="parameter">-nopoll</replaceable>
+      <option>-nopoll</option>
     </arg>
   </group>
   <group choice="opt">
-    <arg choice="opt">
-      <replaceable class="parameter">-label</replaceable>
+    <arg choice="plain">
+      <option>-label</option>
       <replaceable>label</replaceable>
     </arg>
     <arg choice="plain">
-      <replaceable class="parameter">-nolabel</replaceable>
+      <option>-nolabel</option>
     </arg>
   </group>
   <arg choice="plain">
@@ -11091,19 +11091,19 @@ The following are the commands understood by Mutt:
 <arg choice="opt">
   <group choice="opt">
     <arg choice="plain">
-      <replaceable class="parameter">-poll</replaceable>
+      <option>-poll</option>
     </arg>
     <arg choice="plain">
-      <replaceable class="parameter">-nopoll</replaceable>
+      <option>-nopoll</option>
     </arg>
   </group>
   <group choice="opt">
-    <arg choice="opt">
-      <replaceable class="parameter">-label</replaceable>
+    <arg choice="plain">
+      <option>-label</option>
       <replaceable>label</replaceable>
     </arg>
     <arg choice="plain">
-      <replaceable class="parameter">-nolabel</replaceable>
+      <option>-nolabel</option>
     </arg>
   </group>
   <arg choice="plain">
